### PR TITLE
[WIP] Code improvement in the beginner tutorial

### PR DESCRIPTION
**Pull request by ummahusla**
Initially, I was following the beginner tutorial and I've noticed that my code doesn't work. Obviously, later on, I figured out that it was a minor typo I probably removed more code when I supposed to do. I still think that adding `value={store.getState()}` to `<Counter />` will remove that issue and the code will work even if the user is going just simply to copy and paste it.

### CODE REVIEW

**Comment by Andarist** [comment]
Hey,
thanks for the PR - its probably the good idea to include in this tutorial even more of the code involved in real working example - not the `value` props only. It in fact only shows the relevant parts of the code, but there is no indication that there should be more in the final code.

Could you adapt your PR to include all of the `Counter` and `render` code with indication what was added on each step?

**Comment by ummahusla** [comment]
@Andarist Hey, yeah I will think what we can do with that Beginner tutorial and will try to take care of it. Moreover, I've seen that there are issues in the repository for that tutorial. I will evaluate the errors and find anything else we might need to add here.

**Comment by ummahusla** [comment]
@Andarist Hi, can you please review my work. I guess it's ready to be merged :rocket:

File: docs/introduction/BeginnerTutorial.md [hunk 1]
```diff
@@ -247,8 +250,8 @@ Well, `redux-saga` provides a way to make the above statement possible. Instead
 
 ```javascript
 // ...
+import { delay, takeEvery } from 'redux-saga'
```
Comment (Andarist) [inline docs/introduction/BeginnerTutorial.md]: could you import `takeEvery` also from the `'redux-saga/effects'`? Its upcoming change and will be released before the weekend

**Comment by ummahusla** [comment]
@Andarist Thanks 👍

**Pull request merged**